tests/stubtest_allowlists/darwin-py311.txt

```diff
@@ -1,7 +1,8 @@
+# These entries looks like a `setup-python` bug:
 (dbm.gnu)?
-(locale.bind_textdomain_codeset)?
-(locale.bindtextdomain)?
-(locale.dcgettext)?
-(locale.dgettext)?
-(locale.gettext)?
-(locale.textdomain)?
+(_locale.bind_textdomain_codeset)?
+(_locale.bindtextdomain)?
+(_locale.dcgettext)?
+(_locale.dgettext)?
+(_locale.gettext)?
+(_locale.textdomain)?
```

Comment:
It looks like this PR changes things so that these are no longer reexported from `locale.pyi`. Is that deliberate?

If they were being reexported from `locale.pyi`, the allowlist entries would need to be something like

```suggestion
(_?locale.bind_textdomain_codeset)?
(_?locale.bindtextdomain)?
(_?locale.dcgettext)?
(_?locale.dgettext)?
(_?locale.gettext)?
(_?locale.textdomain)?
```